src/events.ts

```diff
@@ -302,7 +302,7 @@ const updateEventResult = (
 export const eventsHandler =
   (context, extrinsicChain: Chain, expectedEvents: Event[], resolve, reject) =>
   async ({ events = [], status }) => {
-    if (status.isReady) {
+    if (status.isInBlock) {
```

Comment:
This was the previous configuration, but I have to change it to `isReady`.

The reason is that if you wait for the extrinsic to be `isInBlock` it can happen that when the event listener is triggered in the remote chain, it is already too late

Reply:
I see